index.md

```diff
@@ -1,70 +1,32 @@
 
-<div style="margin-bottom: 2.25em;">
-  <span style="font-size: 4.85rem; line-height: 1.2; font-weight: 600;">July 13–18, 2018</span><br />
-  <span style="font-size: 3.5rem; line-height: 1.2;">{{ site.title }}</span><br />
-  <h1>Beyond DIY: Do It With Others</h1>
-  <h2>Toronto, Ontario</h2>
-</div>
+What kinds of creative and critical engagement with technology practices can enable meaningful change when we do it with others?
 
-<!-- Event areas section -->
-<section class="sections row featurette-events-row">
-  <div class="four columns">
-    <h3>Kickoff</h3>
-    <span>Friday • 13</span>
-    <p>We open with morning field trips and an afternoon-into-evening social event</p>
-  </div>
-  <div class="four columns">
-    <h3>Conference</h3>
-    <span>Saturday, Sunday • 14, 15</span>
-    <p>The weekend is our main event, 2 days packed full of talks, demos, and workshops</p>
-
-  </div>
-  <div class="four columns">
-    <h3>Sprints</h3>
-    <span>Monday–Wednesday • 16–18</span>
-    <p>The final 3 days are in-person, collaborative project sprints</p>
-  </div>
-</section>
-
-## Conference Call for Participation
-
-<mark><strong>Update</strong>: Deadline extended to Monday, May 7, 2018!</mark>
+Our theme this year focuses on how we **Do It With Others (DIWO)** as opposed to just ourselves, building on the “distributed campaign for emancipatory, networked art practices,” instigated by UK-based Furtherfield (2006) as a response to the Do It Yourself (DIY) movement. We have seen a growing number of peer-to-peer, inclusive, and privacy-respecting projects mobilizing in favour of resilient, accessible, equitable communications over the internet in 2017. And yet many open questions remain. In the face of threats to the open internet, which tools and tactics will help us recognize the opportunities and challenges of this moment?
 
-Our theme this year focuses on how we **Do It With Others (DIWO)** as opposed to just ourselves, building on the "**distributed campaign for emancipatory, networked art practices**," instigated by UK-based [Furtherfield](http://archive.furtherfield.org/projects/diwo-do-it-others-resource) (2006) as a response to the Do It Yourself (DIY) movement. We have seen a growing number of peer-to-peer, inclusive, and privacy-respecting projects mobilizing in favour of resilient, accessible, equitable communications over the internet in 2017. And yet many open questions remain. In the face of threats to the open internet, which tools and tactics will help us recognize the opportunities and challenges of this moment? What kinds of creative and critical engagement with technology practices can enable meaningful change when we do it with others?
+The deadline for conference proposals has now closed, check back late-May for our full conference schedule! In the meantime, are you interested in [selling your stuff](/2018/kickoff-yami-ichi/) at our Internet Yami-ichi? Considering [hosting a project](/2018/sprints/) during our sprints?
```

Comment:
`s/Considering/Consider`? Otherwise it would imply `"interested in" considering`

Reply:
`Sprint` is capitalized in some places. I think they should all be capitalized.

Reply:
Thanks @benhylau -- revised sentence and fixed a couple lower case sprints